src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MvcForm.cs

```diff
@@ -8,11 +8,18 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents an HTML form element in an MVC view.
```

Comment:
Ick "represents". This object represents nothing at all. It's an implementation detail used to ensure we generate one and only one `</form>` end tag.